tests/ord-service/tests/main_test.go

```diff
@@ -34,11 +39,18 @@ var (
 )
 
 type config struct {
-	DirectorURL                   string
-	ORDServiceURL                 string
-	ORDServiceStaticURL           string
-	ORDServiceDefaultResponseType string
-	DefaultScenarioEnabled        bool `envconfig:"default=true"`
+	DirectorURL                      string
+	ORDServiceURL                    string
+	ORDExternalCertSecuredServiceURL string
+	ORDServiceStaticPrefix           string
+	ORDServiceDefaultResponseType    string
+	CACertificate                    []byte `envconfig:"-"`
```

Comment:
Maybe it makes sense now to bunch the cert config into a separate struct and embed it here